x-pack/plugins/apm/public/components/shared/service_icons/icon_popover.tsx

```diff
@@ -22,12 +24,15 @@ interface IconPopoverProps {
   onClose: () => void;
   detailsFetchStatus: FETCH_STATUS;
   isOpen: boolean;
-  icon: {
-    type?: string;
-    size?: 's' | 'm' | 'l';
-    color?: string;
-  };
+  icon: PopoverItem['icon'];
 }
+
+const StyledPanel = styled(EuiPanel)`
+  &.serviceIcon {
+    box-shadow: 0 2px 2px -1px rgba(0, 0, 0, 0.3),
```

Comment:
Could you use `euiShadowColor` here? Also, does it makes sense to copy the rule from `euiBottomShadowSmall` (unless @formgeist is happy with this box-shadow)

Reply:
+1 for using the EUI vars available